docs/environments/jigsaw.md

```diff
@@ -0,0 +1,55 @@
+# Jigsaw Environment
+
+<p align="center">
+        <img src="../env_anim/jigsaw.gif" width="500"/>
+</p>
+
+We provide here a Jax JIT-able implementation of a simple _jigsaw_ puzzle. The goal of the agent is to place
+all the jigsaw pieces in the correct locations on an empty 2D puzzle board. Each time an episode resets a
+new puzzle and set of piece is created. Pieces are randomly shuffled and rotated.
+
+## Observation
+The observation given to the agent gives a view of the current state of the puzzle as well as
+all pieces that can be placed.
+
+- `current_board`: jax array (float32) of shape `(num_rows, num_cols)` with values in the range
+    `[1, num_pieces]` (corresponding to the number of each piece). This board will have zeros
```

Comment:
Board will have zeros where no pieces have been placed. So we need to add 0 to action space

```suggestion
    `[0, num_pieces]` (corresponding to the number of each piece). This board will have zeros
```